ffi/diplomat/Cargo.toml

```diff
@@ -65,12 +65,22 @@ writeable = { version = "0.4", path = "../../utils/writeable/" }
 # Run `cargo make diplomat-install` to get the right diplomat binary installed
 # The version here can either be a `version = ".."` spec or `git = "https://github.com/rust-diplomat/diplomat", rev = ".."`
 # Since this crate is published, Diplomat must be published preceding a new ICU4X release but may use git versions in between
-diplomat = { git = "https://github.com/rust-diplomat/diplomat", rev = "e80b5969a77f9f49e751f63ff51f2af230027c96" }
```

Comment:
Reminder so that we don't forget to change this back to rust-diplomat before merging this PR